ConsumerAPI/DotNet/Program.cs

```diff
@@ -6,74 +6,34 @@
     using System.Text;
     using System.Threading.Tasks;
 
-    using EssentialProjects.IAM.Contracts;
-
-    using ProcessEngine.ConsumerAPI.Client;
-    using ProcessEngine.ConsumerAPI.Contracts;
-    using ProcessEngine.ConsumerAPI.Contracts.DataModel;
+    using ProcessEngineClient;
 
     class Program
     {
-        const string PROCESS_MODEL_ID= "Lager-Manuell";
-        //const string PROCESS_MODEL_ID= "Lager-Teilautomatisch";
-
-
-        const string START_EVENT_ID = "VersandauftragErhalten";
-
-        const string END_EVENT_ID = "VersandauftragVersendet";
-
         static void Main(string[] args)
         {
-            StartProcess().GetAwaiter().GetResult();
+            StartProcessNew().GetAwaiter().GetResult();
         }
 
-        static async Task StartProcess() {
-
-            ConsumerApiClientService client = Program.CreateConsumerClient("http://localhost:8000");
-
-            ProcessStartRequestPayload<StartPayload> payload = Program.CreatePayload("Dies ist die Eingabe für den Prozess aus DotNet.");
-
-            IIdentity identity = CreateIdentity();
+        static async Task StartProcessNew() 

```

Comment:
```suggestion
        private static async Task StartNewProcessInstance() 
```

Üblicherweise reden wir immer von `ProcessModel`, wenn es um Modelle geht und `ProcessInstance`, wenn es um ausgeführte Prozessinstanzen geht.